spec/controllers/billing_subscriptions_spec.rb

```diff
@@ -11,7 +11,7 @@
     end
 
     describe 'cancellation' do
-        include_context :open_to_np_admin, :get, :cancellation, nonprofit_id: :__our_np
+        include_context :open_to_np_admin, :get, :cancellation, nonprofit_id: :__our_np, without_json_view: true
```

Comment:
`open_to_np_admin` checks the authorization for a given set a users. In this case, it's authorized for np_admins and higher but not lower. On success, in Rails 4, this action returned the same regardless of whether the caller requested application/json or not. In Rails 5, this crashed. So this tells the test "hey, don't ask for `application/json`.